audio.h

```diff
@@ -3,4 +3,5 @@
 void audioInit(void);
 void audioStart(void);
 void audioDeInit(void);
-int16_t* audioGetMicData(void);
\ No newline at end of file
+int16_t* audioGetMicData(void);
+void decimate(int16_t *buffer, uint8_t dec_factor);
```

Comment:
looks like this is a function exclusive to audio.c. remove from the header and make the definition in audio.c static